services/graphql/src/graphql/resolvers/access-level.js

```diff
@@ -35,21 +35,22 @@ module.exports = {
     /**
      *
      */
-    matchAccessLevels: (_, { input }, { app }) => {
+    matchAccessLevels: async (_, { input }, { app }) => {
```

Comment:
You might as well convert this to use a `AccessLevelConnection` so it's consistent with everything else.